logfire-api/logfire_api/_internal/tracer.pyi

```diff
@@ -33,10 +36,11 @@ class ProxyTracerProvider(TracerProvider):
     def force_flush(self, timeout_millis: int = 30000) -> bool: ...
 
 @dataclass
-class _MaybeDeterministicTimestampSpan(trace_api.Span, ReadableSpan):
+class _LogfireWrappedSpan(trace_api.Span, ReadableSpan):
```

Comment:
Rename makes sense 👍🏻 

Should we make it a `__slots__` data class (we should do it for all of them), using dynamic kwargs for python < 3.10, etc.? Can we another PR

Reply:
I'd rather wait until we finish the refactoring to make it a `__slots__` dataclass, just in case. But I think that makes sense once things are stable, maybe it would be fine to do it now I'd just rather not be thinking about it _yet_. Can change it now if you insist though.